src/search/FindInFiles.js

```diff
@@ -417,15 +418,26 @@ define(function (require, exports, module) {
         }
         
         var scopeName = searchModel.scope ? searchModel.scope.fullPath : ProjectManager.getProjectRoot().fullPath,
-            perfTimer = PerfUtils.markStart("FindIn: " + scopeName + " - " + queryInfo.query);
+            perfTimer = PerfUtils.markStart("FindIn: " + scopeName + " - " + queryInfo.query),
+            searchDeferred = new $.Deferred();
         
-        return candidateFilesPromise
+        candidateFilesPromise
             .then(function (fileListResult) {
                 // Filter out files/folders that match user's current exclusion filter
                 fileListResult = FileFilters.filterFileList(filter, fileListResult);
+                fileListResult = FindUtils.getSortedFiles(fileListResult, function (file) { return file.fullPath; }, FindUtils.getOpenFilePath());
```

Comment:
@TomMalbran I've been trying to speed up this code and I discovered that the initial call to `FindUtils.getSortedFiles()` here that you added is taking a lot of time -- ~3000ms in [the scenario I use here](https://github.com/adobe/brackets/pull/11128#issuecomment-116751074). Even worse, this is before panel is displayed the first time, so there's a 3 sec gap with no ui feedback!

I think that the files are all correctly sorted *except* that we want to move current doc to the top of the list. Is that correct? If so, we could write a separate function to do that much faster.

Let me know what you think. I think this is the main blocker to get this into master.